tools/tick-cluster.js

```diff
@@ -517,6 +536,7 @@ function main() {
 function displayMenu(logFn) {
     logFn('\td <flag>\tSet debug flag');
     logFn('\tD\t\tClear debug flags');
+    logFn('\tG\t\tStop gossip');
```

Comment:
the other commands are listing the lowercase option before the uppercase one. Can you follow that style here?